src/useKeyboardEvent/useKeyboardEvent.ts

```diff
@@ -0,0 +1,38 @@
+import type { Key } from 'ts-key-enum';
+import { useEventListener } from '../useEventListener/useEventListener';
+import type { EventTarget } from '../useEventListener/useEventListener';
+
+export type KeyDownCallback = (event: GlobalEventHandlersEventMap['keydown']) => void;
+
+/**
+ * The Key enum contains values for all standard non-printable keys such as "CapsLock", "Backspace",
+ * and "AudioVolumeMute". The enum does not contain values for printable keys such as "a", "A", "#",
+ * "é", or "¿", simply because the list of possible values is too vast to include in a single enum.
+ * To test for printable values, simply use a string comparison:
+ *
+ * See -> https://gitlab.com/nfriend/ts-key-enum#whats-included
+ */
+export type KeyboardEventKey = Key | string;
```

Comment:
I don't understand this type. We are specifying that `Key` (from `ts-key-enum` is the type, but then we are making it "loose" with `| string`, so the type we end up with is `string`.

So then what is the point of adding `Key` here ?

Reply:
We should either commit to using `ts-key-enum` or specify `string` to match `dom.d.ts`. Doing the latter we could still encourage `ts-key-enum`, as it does feel like a convenience rather than a hard requirement.

Reply:
Agree, we should type as `string` (as allowing only `Key` is too constrained).

Using the `ts-key-enum` is encouraged (and we might want to include that in the docs for convenience), but definitely not required.

Reply:
Done!